packages/sanes-chrome-extension/src/routes/account/index.tsx

```diff
@@ -59,7 +59,7 @@ const AccountView = (): JSX.Element => {
 
   return (
     <Drawer items={items}>
-      <PageLayout id={ACCOUNT_STATUS_ROUTE} primaryTitle="Account" title="Status">
+      <PageLayout id={ACCOUNT_STATUS_ROUTE} primaryTitle="Account" title="Status" minHeight="444px">
```

Comment:
Why 444px?

Reply:
500-56 because the Drawer is 56px high.

Reply:
The extract to a constant the Drawer height, place it inside Drawer and make the calculation, something like:
```
const MAIN_LAYOUT = CHROME_EXTENSION_HEIGHT - DRAWER_HEIGHT
```